src/signals/incident-management/components/FilterForm/index.js

```diff
@@ -243,6 +251,19 @@ const FilterForm = ({ filter, onCancel, onClearFilter, onSaveFilter, onSubmit, o
           </div>
         </Fieldset>
 
+        <Fieldset>
+          <FilterGroup>
+            <Label htmlFor="filter_notes" isGroupHeader>Zoek in notitie</Label>
+            <Input
+              name="note_keyword"
+              id="filter_notes"
```

Comment:
Why not use camel casing?

Reply:
Because it would be inconsistent with the other naming in this file, and it is also related to the backend which does not use camel casing since it's written in Python.

Reply:
Besides that camel case naming is not common for HTML element attributes, I would prefer `hyphens` ;)

Reply:
I see. I would map backend snake casings to camel casing as soon as the data comes in, but that's out of scope of this PR of course.
In HTML I would still use camel casing, since they'll come back in you javascript code.